Sources/Holocron/FilePersistence.swift

```diff
@@ -6,6 +6,8 @@
 //  Copyright © 2017 Bottle Rocket. All rights reserved.
 //
 
+// Testing Danger...
```

Comment:
did you intentionally want this line here?

Reply:
Whoops, forgot to reverse that commit. Removed it now!